contracts/TokenConverter/RiskFundConverter.sol

```diff
@@ -238,8 +238,8 @@ contract RiskFundConverter is AbstractTokenConverter {
         for (uint256 i; i < poolsLength; ++i) {
             uint256 poolShare = (poolsAssetsReserves[pools[i]][tokenOutAddress] * EXP_SCALE) / assetReserve;
             if (poolShare == 0) continue;
```

Comment:
Would it make sense this to save gas?

1. save `poolsAssetsReserves[pools[i]][tokenOutAddress]` in a temporary variable. Let's call it `currentPoolsAssetsReserves`
2. add `if (currentPoolsAssetsReserves == 0) continue;`, instead of checking the `poolShare` variable, that could be removed

Reply:
https://github.com/VenusProtocol/protocol-reserve/pull/29/commits/7f289d05c6993f8c24dea0fbeba536df33d428fd